make/release.mk

```diff
@@ -1,26 +1,39 @@
-IMAGE_BUILD_DIR := _build
-IMAGE_TAR_FILE := $(IMAGE_BUILD_DIR)/catalog-applications-image-bundle.tar.gz
-CATALOG_IMAGES_TXT := $(IMAGE_BUILD_DIR)/catalog_images.txt
+BUILD_DIR := _build
+IMAGE_TAR_FILE := $(BUILD_DIR)/catalog-applications-image-bundle.tar.gz
+REPO_ARCHIVE_FILE := $(BUILD_DIR)/dkp-catalog-applications.tar.gz
+CATALOG_IMAGES_TXT := $(BUILD_DIR)/catalog_images.txt
```

Comment:
same here maybe?

Reply:
yep done in 61b36c138e6a0920e6e3b265a1b4d966ce957e0b as well